config/zones.json

```diff
@@ -2115,28 +3902,28 @@
     ],
     "flag_file_name": "us.png",
     "parsers": {
-      "production": "US_NEISO.fetch_production",
-      "consumptionForecast": "EIA.fetch_consumption_forecast"
+      "consumptionForecast": "EIA.fetch_consumption_forecast",
+      "production": "US_NEISO.fetch_production"
     },
     "timezone": null
   },
   "US-NY": {
     "_comment": "hydro incl. 1407 MW pumped storage",
     "capacity": {
-      "gas": 22117,
       "coal": 1011,
-      "oil": 2499,
-      "nuclear": 5375,
+      "gas": 22117,
       "hydro": 5658,
+      "nuclear": 5375,
+      "oil": 2499,
       "wind": 1740
     },
     "contributors": [
       "https://github.com/systemcatch"
     ],
     "flag_file_name": "us.png",
     "parsers": {
-      "production": "US_NY.fetch_production",
-      "consumptionForecast": "EIA.fetch_consumption_forecast"
+      "consumptionForecast": "EIA.fetch_consumption_forecast",
```

Comment:
Seems like you need to merge the latest master onto your branch?
This chance shouldn't be there I think.

Reply:
indeed! Tricky github